### DISCUSSION TITLE
org.infinispan.protostream.DescriptorParserException: Import 'java.util.proto' not found"

### DISCUSSION THREAD

**Discussion by MelangaK97** [infinispan/infinispan#12162] [Q&A]
Hi,
Any suggestion to avoid this error? Occurred while upgrading the infinispan version from 11 to 14.
**Caused by: org.infinispan.commons.CacheConfigurationException: Error starting component org.infinispan.marshall.protostream.impl.SerializationContextRegistry
    Caused by: org.infinispan.protostream.DescriptorParserException: Import 'java.util.proto' not found"**

**Comment by tristantarrant** [comment]
Sorry for missing this: there is now such thing as `java.util.proto` so we would need to see your protobuf definitions
